sonic_platform_base/sonic_sfp/sfputilhelper.py

```diff
@@ -46,6 +52,30 @@ def read_porttab_mappings(self, porttabfile, asic_inst=0):
         port_pos_in_file = 0
         parse_fmt_port_config_ini = False
 
+        if get_ports_from_db:
+            (platform, hwsku) = device_info.get_platform_and_hwsku()
+            (ports, _, _) = get_port_config(hwsku, platform, get_ports_from_db=True)
+            if not ports:
+                helper_logger.log_error("failed to get ports info from config DB")
```

Comment:
should 'return' here if it failed to get the ports data, we don't want to get to the loop in such case

Reply:
if ports is empty ports.keys() returns none and we will not get into the for.
I can add a return but it will not have any meaning.

Reply:
Done.